metric/system/process/testdata/winbinaries/ProcessSnapshotting.cpp

```diff
@@ -0,0 +1,168 @@
+// from https://github.com/zodiacon/Win10SysProgBookSamples/blob/56883f5126f5e29a03d01896f87ce7b82f51fa10/Chapter20/snapproc/snapproc.cpp
```

Comment:
What is the license of this code? Are you allowed to copy it here?

It seems like you might be better off with just linking to the book with instructions on how to build it unless you've modified this significantly.